Kickstarter-iOS/Views/Storyboards/Settings.storyboard

```diff
@@ -780,30 +780,30 @@
                                                 <constraints>
                                                     <constraint firstItem="1Rl-Tz-d27" firstAttribute="leading" secondItem="CrV-4u-w6R" secondAttribute="leading" id="0bT-q6-KkH"/>
                                                     <constraint firstAttribute="bottom" secondItem="lmn-J1-wDV" secondAttribute="bottom" id="9aI-sL-UB6"/>
+                                                    <constraint firstAttribute="trailing" secondItem="lmn-J1-wDV" secondAttribute="trailing" constant="12" id="dxS-vb-uUL"/>
                                                     <constraint firstItem="1Rl-Tz-d27" firstAttribute="top" secondItem="CrV-4u-w6R" secondAttribute="top" id="fPZ-aM-ygc"/>
-                                                    <constraint firstAttribute="trailing" secondItem="1Rl-Tz-d27" secondAttribute="trailingMargin" id="law-tX-KFD"/>
```

Comment:
Just want to double check, were these changes related to the `Edit` button?

Reply:
I think this diff showed up during a rebase or something. I didn't touch the `Settings.storyboard`